project/Version.scala

```diff
@@ -21,7 +21,7 @@ object Version {
     Properties.envOrElse(environmentVariable, default)
 
   val geotrellis  = "0.10.0-SNAPSHOT"
-  val scala       = "2.11.5"
```

Comment:
Ok, that makes sense. Can you leave a comment in the build file talking about why we are using the lower version as the default?